cves/kernel/CVE-2017-18360.yml

```diff
@@ -149,10 +159,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
```

Comment:
Perhaps add some evidence from commits or other comments about where/who discovered this.

Reply:
Added the evidence